hail/python/hailtop/batch/job.py

```diff
@@ -539,6 +539,11 @@ def handler(match_obj):
                         raise BatchException(f"undefined resource '{name}'\n"
                                              f"Hint: resources must be defined within "
                                              f"the job methods 'command' or 'declare_resource_group'")
+
+                    if self._always_run:
+                        warnings.warn('A job marked as always run has a resource file dependency on another job. If the dependent job fails, '
+                                      f'the always run job with the following command will not run:\n{command}')
```

Comment:
I think this should be `the always run job with the following command may not succeed`. It *will* run, right?